app/models/user_track.rb

```diff
@@ -19,6 +19,8 @@ class UserTrack < ApplicationRecord
     primary_key: :user_id,
     inverse_of: :user_track
 
+  delegate :num_concepts, to: :track
```

Comment:
I've decided to also delegate to `Track` in `UserTrack` to ensure that the logged-in and logged-out values are consistent.